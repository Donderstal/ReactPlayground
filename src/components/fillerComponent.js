import * as React from 'react'


class fillerComponent extends React.Component {
    constructor(props) {
        super(props);
    }

    fillerFunction(num1, num2) {
        return (num1 / num2)
    }

    render(){
        return(
             <Text>Hey!</Text>
        )
    }
}

export default fillerComponent 