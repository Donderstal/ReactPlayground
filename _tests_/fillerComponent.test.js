import React from 'react';
import renderer from 'react-test-renderer';
import * as Adapter from 'enzyme-adapter-react-16';
import Enzyme, { shallow, configure } from 'enzyme';


import fillerComponent from '../src/components/fillerComponent';

configure({ adapter: new Adapter() });


test('testing fillerComponent...', () => {
     const tree = renderer.create(<fillerComponent />).toJSON();
      expect(tree).toMatchSnapshot();
});

describe('<fillerComponent> functions', () => {
    it('should be as I want it to', () => {
        const wrapper = shallow(<fillerComponent />);
        expect(wrapper.instance().fillerFunction(1 , )).equals(0.5);
    })
})