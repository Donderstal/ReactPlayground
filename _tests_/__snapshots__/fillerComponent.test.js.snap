// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing fillerComponent... 1`] = `<fillerComponent />`;
